spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -71,3 +61,23 @@ Then obtain a `CompletableFuture` or `CompletionStage` from the `ManagedExecutor
                                              .thenApplyAsync(function1)
                                              .thenApply(function2);
 ----
+
+Take care to shut down managed executors once the application no longer needs them:
```

Comment:
Do you think it might be worth to show the disposer method as well? Not necessarily in this file.

Reply:
Sure, I can add an example disposer